x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/application/EnterpriseSearchFeatureSetUsageSerializingTests.java

```diff
@@ -19,19 +19,25 @@ public class EnterpriseSearchFeatureSetUsageSerializingTests extends AbstractWir
     @Override
     protected EnterpriseSearchFeatureSetUsage createTestInstance() {
         Map<String, Object> searchApplicationsStats = new HashMap<>();
+        Map<String, Object> analyticsCollectionsStats = new HashMap<>();
         searchApplicationsStats.put("count", randomLongBetween(0, 100000));
-        return new EnterpriseSearchFeatureSetUsage(true, true, searchApplicationsStats);
+        analyticsCollectionsStats.put("count", randomLongBetween(0, 100000));
+        return new EnterpriseSearchFeatureSetUsage(true, true, searchApplicationsStats, analyticsCollectionsStats);
     }
 
     @Override
     protected EnterpriseSearchFeatureSetUsage mutateInstance(EnterpriseSearchFeatureSetUsage instance) throws IOException {
-        long searchApplicationsCount = (long) instance.getSearchApplicationsStats().get("count");
+        long searchApplicationsCount = (long) instance.getSearchApplicationsUsage().get("count");
```

Comment:
Would it be worth moving some of the hard coded strings like `"count"` and moving them into variables we could reference?